pkg/workloads/cortex/lib/type/model.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 Cortex Labs, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+class Model:
+    def __init__(self, name=None, model=None, signature_key=None, base_path=None):
```

Comment:
In what scenario would either name, model or base_path be None? Maybe they should be required positional args?

Reply:
None that I can think of. I'm making them positional arguments.